### PR TITLE
Clean up and test camerad refactor

### PR DESCRIPTION
**Pull request by ZwX1616**
#2125 

Did some final cleanups, and move here to run ci tests

tici jpgs fixed as well, which is nice

### CODE REVIEW

File: selfdrive/camerad/cameras/camera_common.cc [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+    callback(cameras, cs, cnt);
+
+    cs->buf.release();
+  }
+  return NULL;
+}
+
+std::thread start_process_thread(MultiCameraState *cameras, const char *tname,
+                                          CameraState *cs, int priority, process_thread_cb callback) {
```
Comment (adeebshihadeh) [inline selfdrive/camerad/cameras/camera_common.cc:262]: spacing

File: selfdrive/camerad/cameras/camera_common.cc [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+#elif QCOM2
+#include "cameras/camera_qcom2.h"
+#elif WEBCAM
+#include "cameras/camera_webcam.h"
+#else
+#include "cameras/camera_frame_stream.h"
+#endif
+
+#include "camera_common.h"
+#include <czmq.h>
```
Comment (adeebshihadeh) [inline selfdrive/camerad/cameras/camera_common.cc:16]: why is zmq needed?

Comment (ZwX1616) [reply selfdrive/camerad/cameras/camera_common.cc:16]: only for tici debugging and fl I think? may move to qcom2.cc

Comment (ZwX1616) [reply selfdrive/camerad/cameras/camera_common.cc:16]: oh nvm, qcom also uses zmq to do exposure/focus and stuff

**Comment by ZwX1616** [comment]
TO FIX:

- [ ] thumbnails broken?

- [ ] init exposure params correctly

**Comment by ZwX1616** [comment]
- [ ] 1 

- [ ] 1L

- [ ] 2

- [ ] 2L

- [ ] 3

- [ ] 3L

File: selfdrive/camerad/main.cc [hunk 3]
```diff
@@ -60,7 +60,7 @@ static CameraBuf *get_camerabuf_by_type(VisionState *s, VisionStreamType type) {
   assert(type >= 0 && type < VISION_STREAM_MAX);
   if (type == VISION_STREAM_RGB_BACK || type == VISION_STREAM_YUV) {
     return &s->cameras.rear.buf;
-  } else if (type == VISION_STREAM_RGB_FRONT || VISION_STREAM_YUV_FRONT) {
+  } else if (type == VISION_STREAM_RGB_FRONT || type == VISION_STREAM_YUV_FRONT) {
```
Comment (deanlee) [inline selfdrive/camerad/main.cc:63]: lol,omg

**Pull request merged**